test/SmokeTests/BackendSmokeTests/Netherite/Netherite.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\src\WebJobs.Extensions.DurableTask\WebJobs.Extensions.DurableTask.csproj" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Netherite.AzureFunctions" Version="1.4.0" />
```

Comment:
is it possible to pick up the latest Netherite version here, as opposed to the specific version 1.4.0?
Otherwise we will have to remember to update this every once in a while.

Reply:
Trying to do that here: https://github.com/Azure/azure-functions-durable-extension/pull/2597/commits/9db26787f2aae5053f23215c2f37d83c7e52835c

Although I see some tests broke as a result. Let me look into that.

Reply:
Seems to be working now :)